tests/CMakeLists.txt

```diff
@@ -203,10 +211,10 @@ foreach(target ${test_targets})
 
   # Always write the output file directly into the 'tests' directory (even on MSVC)
   if(NOT CMAKE_LIBRARY_OUTPUT_DIRECTORY)
-    set_target_properties(${target} PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${testdir})
+    set_target_properties(${target} PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${testdir}")
```

Comment:
I thought cmake doesn't care about quotes.

Reply:
No, without quotes spaces turn into separate arguments. All filename variables need to have quotes. We aren't testing with a space in the name of a directory, but I expect this would fail without quotes.